cli/cmdlineparser.cpp

```diff
@@ -941,8 +941,10 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
             }
 
             // Write results in results.xml
-            else if (std::strcmp(argv[i], "--xml") == 0)
+            else if (std::strcmp(argv[i], "--xml") == 0) {
+                printMessage("option --xml is deprecated; use --output-format=xml instead.");
                 mSettings->xml = true;
```

Comment:
I suggest we remove variable xml in the Settings. And create enum outputFormat instead.